src/core/url_store/atoms/urlStore.ts

```diff
@@ -99,7 +107,7 @@ export const urlStoreAtom = createAtom(
 
     /* After initialization finished - write new changes from state back to url */
     const newState = { ...state };
-    const currentMapPosition = get('currentMapPositionAtom');
+    const currentMapPosition = get('currentMapPositionAtomV2');
```

Comment:
Not sure how to properly replace this v2 atom dependency with v3 atom:
`const currentMapPosition = get('currentMapPositionAtomV2');`

@vkozio @dqunbp Maybe you can help here?

Reply:
Slack discussion: https://konturio.slack.com/archives/C02DEKP2MKP/p1732786548771199

Reply:
Fixed